nextstrain/cli/remote/nextstrain_dot_org.py

```diff
@@ -629,6 +651,19 @@ def raise_for_status(response: requests.Response) -> None:
                 Check for typos in the parameters you used?
                 """) from err
 
+        elif status in range(500, 600):
+            raise UserError(f"""
+                The remote server had a problem processing our request.
+
+                Retrying may help if the problem happens to be transient, or
+                there might be a bug somewhere that needs to be fixed.
+
+                If retrying after a bit doesn't help, please open a new issue
+                at <https://github.com/nextstrain/cli/issues/new/choose> and
+                include the complete output above and the command you were
+                running.
+                """) from err
```

Comment:
Agreed. I had a similar thought, and my (tentative) conclusion was that instead of refactoring the whole error message, we could in the future refactor just the bit about reporting an issue. This would let us keep that centralized while still letting the rest of the error messages include local context.